source/installing.rst

```diff
@@ -0,0 +1,214 @@
+
+Prerequisites
+================================================================================
+
+Install |percona-release| if you haven't done it already. Otherwise, update it to the latest version. Please refer to `Percona repositories documentation <https://www.percona.com/doc/percona-repo-config/percona-release.html>`_ for installation or update instructions.
+
+Enable Percona repository       
+================================================================================
+
+We recommend to use the ``setup`` subcommand of |percona-release| to enable the desired repository.
+ 
+* For |PS|-based deployment, use the following command:
+
+  .. code-block:: bash
+
+     $ sudo percona-release setup pdps-8.0
+
+* For |PXC|-based deployment, use the following command:
+
+  .. code-block:: bash
+
+     $ sudo percona-release setup pdpxc-8.0
+
+Make sure to run |percona-release| as root or via |sudo|. For the
+sake of convenience, all commands that require elevated privileges start with
+|sudo| in the following procedures.
+
+Install |pdmysql|
+================================================================================
+
+Install the selected deployment of |pdmysql| using the commands of the package manager of your operating system.
+
+.. important::
+
+   Make sure to run all commands as root or via |sudo|.
+
+Install |PS|-based deployment on Debian / Ubuntu
+--------------------------------------------------------------------------------
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo apt-get install percona-server
+
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Percona XtraBackup
+      $ sudo apt-get install percona-xtrabackup-80
+      $ #Install Percona Toolkit
+      $ sudo apt-get install percona-toolkit
+      $ #Install Orchestrator
+      $ sudo apt-get install percona-orchestrator
+      $ #Install MySQL Shell
+      $ sudo apt-get install percona-mysql-shell
+      $ #Install ProxySQL
+      $ sudo apt-get install proxysql2
+      $ #Install MySQL Router
+      $ sudo apt-get install percona-mysql-router  
+      
+.. seealso::
+
+   Percona Documentation:
+       * `Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation/apt_repo.html>`_
+       * `Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation/apt_repo.html>`_
+       * `Percona Toolkit <https://www.percona.com/doc/percona-toolkit/2.2/installation.html>`_
+ 
+
+Install |PS|-based deployment on |RHEL| / |centos| 
+--------------------------------------------------------------------------------
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo yum install percona-server
+
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Orchestrator
+      $ sudo yum install percona-orchestrator
```

Comment:
need to add note that epel-release package should be installed as 2 orchestrator dependencies (jq and oniguruma) should be installed from it

Reply:
Is it appllicable for all platforms or for Centos 7, like in Postgresql docs?

Reply:
Yes it is needed only on centos7. On centos8 packages would be installed from default AppStream repo.